src/app/components/TransactionsTable/index.tsx

```diff
@@ -24,46 +24,42 @@ type Transaction = {
 export default function TransactionsTable({ transactions }: Props) {
   function renderIcon(type: string) {
     function getIcon() {
-      const iconClasses = "h-4 w-4";
+      const iconClasses = "h-3 w-3";
       switch (type) {
         case "received":
           return <PlusIcon className={iconClasses} aria-hidden="true" />;
         case "sent":
-          return <MinusIcon className={iconClasses} aria-hidden="true" />;
         case "sending":
           return <MinusIcon className={iconClasses} aria-hidden="true" />;
       }
     }
 
     return (
-      <div className="flex justify-center items-center w-7 h-7 border-2 border-gray-200 rounded-full">
+      <div className="flex justify-center items-center w-6 h-6 border-2 border-gray-200 rounded-full">
         {getIcon()}
       </div>
     );
   }
 
   return (
-    <div className="shadow overflow-hidden border-b border-gray-200 rounded-lg">
+    <div className="shadow overflow-hidden rounded-lg">
       <div className="bg-white divide-y divide-gray-200">
         {transactions.map((tx) => (
           <div className="px-3 py-2">
             <Disclosure>
               {({ open }) => (
-                <div key={tx.id} className="flex justify-between space-x-5">
-                  <div className="overflow-hidden">
-                    <div className="flex mt-2">
-                      <div className="flex-shrink-0 mr-3">
-                        {tx.type && renderIcon(tx.type)}
-                      </div>
-                      <div className="overflow-hidden">
-                        <div className="text-gray-900 truncate">{tx.title}</div>
-                        <Disclosure.Panel>
-                          <div className="text-sm text-gray-500 truncate">
-                            {tx.subTitle}
-                            <p className="truncate">Preimage: {tx.preimage}</p>
-                          </div>
-                        </Disclosure.Panel>
+                <>
+                  <div key={tx.id} className="flex">
+                    <div className="flex items-center flex-shrink-0 mr-3">
+                      {tx.type && renderIcon(tx.type)}
+                    </div>
+                    <div className="overflow-hidden mr-3">
+                      <div className="text-sm font-medium text-gray-900 truncate">
+                        {tx.title}
```

Comment:
where do we set the link for this? can we link directly to the publishers detail page? (right now we link to the publishers list.

but I guess in the parent component where we use the TransactionTable we do not know the link of the individual payment and thus only have one link for all..?

Reply:
We can link to: options.html/publishers/${id}

Reply:
ah true, we can do that here? https://github.com/getAlby/lightning-browser-extension/blob/master/src/app/screens/Home/index.jsx#L154 correct?

Reply:
I updated the links for an allowance. When there is no allowance (just payments), I'm not sure yet how we can get the publisher id.